models/webhook.go

```diff
@@ -0,0 +1,68 @@
+package models
+
+import (
+  "errors"
+
+  log "github.com/gophish/gophish/logger"
+)
+
```

Comment:
Please add comments for all exported functions and variables in this file. Running `golint` will help you out here.